x-pack/packages/kbn-elastic-assistant/impl/assistant/settings/assistant_settings_management.tsx

```diff
@@ -0,0 +1,340 @@
+        setSelectedConversation(conversationSettings[selectedConversation.title]);
+      }
+    }, [conversationSettings, selectedConversation]);
+
+    // Quick Prompt Selection State
+    const [selectedQuickPrompt, setSelectedQuickPrompt] = useState<QuickPrompt | undefined>();
+    const onHandleSelectedQuickPromptChange = useCallback((quickPrompt?: QuickPrompt) => {
+      setSelectedQuickPrompt(quickPrompt);
+    }, []);
+    useEffect(() => {
+      if (selectedQuickPrompt != null) {
+        setSelectedQuickPrompt(
+          quickPromptSettings.find((q) => q.title === selectedQuickPrompt.title)
+        );
+      }
+    }, [quickPromptSettings, selectedQuickPrompt]);
+
+    // System Prompt Selection State
+    const [selectedSystemPrompt, setSelectedSystemPrompt] = useState<Prompt | undefined>();
+    const onHandleSelectedSystemPromptChange = useCallback((systemPrompt?: Prompt) => {
+      setSelectedSystemPrompt(systemPrompt);
+    }, []);
+    useEffect(() => {
+      if (selectedSystemPrompt != null) {
+        setSelectedSystemPrompt(systemPromptSettings.find((p) => p.id === selectedSystemPrompt.id));
+      }
+    }, [selectedSystemPrompt, systemPromptSettings]);
+
+    const handleSave = useCallback(() => {
+      // If the selected conversation is deleted, we need to select a new conversation to prevent a crash creating a conversation that already exists
+      const isSelectedConversationDeleted =
+        conversationSettings[defaultSelectedConversation.title] == null;
+      const newSelectedConversationId: string | undefined = Object.keys(conversationSettings)[0];
+      if (isSelectedConversationDeleted && newSelectedConversationId != null) {
+        setSelectedConversationId(conversationSettings[newSelectedConversationId].title);
+      }
+      saveSettings();
+      toasts?.addSuccess({
+        iconType: 'check',
+        title: i18n.SETTINGS_UPDATED_TOAST_TITLE,
+      });
+    }, [
+      conversationSettings,
+      defaultSelectedConversation.title,
+      saveSettings,
+      setSelectedConversationId,
+      toasts,
+    ]);
+
+    const tabsConfig = useMemo(
+      () => [
+        {
+          id: CONVERSATIONS_TAB,
+          label: i18n.CONVERSATIONS_MENU_ITEM,
+          prepend: <EuiIcon type="discuss" />,
+        },
+        {
+          id: QUICK_PROMPTS_TAB,
+          label: i18n.QUICK_PROMPTS_MENU_ITEM,
+          prepend: <EuiIcon type="editorComment" />,
+        },
+        {
+          id: SYSTEM_PROMPTS_TAB,
+          label: i18n.SYSTEM_PROMPTS_MENU_ITEM,
+          prepend: <EuiIcon type="editorComment" />,
+        },
+        {
+          id: ANONYMIZATION_TAB,
+          label: i18n.ANONYMIZATION_MENU_ITEM,
+          prepend: <EuiIcon type="eyeClosed" />,
+        },
+        {
+          id: KNOWLEDGE_BASE_TAB,
+          label: i18n.KNOWLEDGE_BASE_MENU_ITEM,
+          prepend: <EuiIcon type="notebookApp" />,
+        },
+        ...(modelEvaluatorEnabled
```

Comment:
Hmmm, I'm not seeing the evaluator tab even though I have `modelEvaluatorEnabled` set...


Ahhh, looks like this is a plugin feature registration issue. Features are registered on a per-plugin basis, and so `assistantModelEvaluation` is only getting registered for the `securitySolutionUI` plugin.

https://github.com/elastic/kibana/blob/04c21e6ca95b5b93efa63ae48f41051075e6158f/x-pack/plugins/security_solution/server/plugin.ts#L558-L562

And when the `get_capabilities` route is hit from the management page, it is using `management` as the `pluginName` and so doesn't find the evaluator.


https://github.com/elastic/kibana/blob/04c21e6ca95b5b93efa63ae48f41051075e6158f/x-pack/plugins/elastic_assistant/server/routes/capabilities/get_capabilities_route.ts#L54-L59


---

I think the straightforward fix here is to just register the same features for the `management` plugin as well, but that kinda breaks the intended 'separation of concern' ability of registering different features. I don't think this is a big deal at the moment, and we can re-visit this whenever we have the need for separation, so probably just easiest to add this below the other registration in `security_solution/server/plugin.ts`:

``` ts
plugins.elasticAssistant.registerFeatures('management, {
  assistantModelEvaluation: config.experimentalFeatures.assistantModelEvaluation,
});
```

Reply:
This looks good -- ty!

<img width="1296" alt="image" src="https://github.com/elastic/kibana/assets/2946766/653560f1-6a7d-471a-82d3-5d8f7ce4b452">